core/src/main/java/com/softwaremill/kmq/KmqConfig.java

```diff
@@ -90,4 +106,29 @@ public String getRedeliveryCountHeader() {
     public int getMaxRedeliveryCount() {
         return maxRedeliveryCount;
     }
+
+    public String getBootstrapServers() {
+        return bootstrapServers;
+    }
+
+    public Map<String, String> getProducerProps() {
+        Map<String, String> props = new HashMap<>();
+        props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
+        props.put(ProducerConfig.ACKS_CONFIG, "all");
+        props.put(ProducerConfig.RETRIES_CONFIG, "0");
+        props.put(ProducerConfig.BATCH_SIZE_CONFIG, "16384");
+        props.put(ProducerConfig.LINGER_MS_CONFIG, "1");
+        props.put(ProducerConfig.BUFFER_MEMORY_CONFIG, "33554432");
```

Comment:
I'm wondering if the above three config options are needed - no idea why they are explicitly set (batch size, linger, buffer memory). Maybe try removing them and see if things still work?

Reply:
For the record: 
- BATCH_SIZE and LINGER_MS - probably improve overall performance (fewer small messages sent to Kafka at a cost of 1ms latency)
- BUFFER_MEMORY - improves safety adding a specific upper bound on message size allowed.

Currently these settings (ProducerConfig) are used only in the Redeliverer so it's the same as before. 
I personally would keep them. What do you think?

Reply:
ok :)